Exceptions/SchemaValidationException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ValidationBundle\Exceptions;
+
+use Exception;
+
+class SchemaValidationException extends Exception
```

Comment:
phpdoc and try to extend from `Symfony\Component\HttpKernel\Exception\BadRequestHttpException`